examples/composite_queries/src/canister1/canister1.py

```diff
@@ -0,0 +1,160 @@
+from kybra import Async, CanisterResult, nat, Principal, query, update, manual, ic
+from src.canister1.types import Canister1, NatQueryResult, StringQueryResult
+from src.canister2.types import Canister2
+
+
+canister1 = Canister1(Principal.from_str('rrkah-fqaaa-aaaaa-aaaaq-cai'))
+canister2 = Canister2(Principal.from_str('ryjl3-tyaaa-aaaaa-aaaba-cai'))
+counter: nat = 0
+
+
+# Composite query calling a query
+@query
+def simple_composite_query() -> Async[StringQueryResult]:
+    result: CanisterResult[str] = yield canister2.simple_query()
+
+    if result.err is not None:
+        return {
+            'err': result.err
+        }
+
+    return {
+        'ok': result.ok
+    }
+
+
+# Composite query calling a manual query
+@query
+def manual_query() -> Async[StringQueryResult]:
+    result: CanisterResult[str] = yield canister2.manual_query()
+
+    if result.err is not None:
+        return {
+            'err': result.err
+        }
+
+    return {
+        'ok': result.ok
+    }
+
+
+# Manual composite query calling a manual query
+@query
+def totally_manual_query() -> Async[manual[StringQueryResult]]:
+    result: CanisterResult[str] = yield canister2.manual_query()
+
+    if result.err is not None:
```

Comment:
Why does Azle look so different? In Azle we are returning without unwrapping the result...why is that?